atlassian/bitbucket.py

```diff
@@ -183,30 +176,32 @@ def repo_keys(self, project_key, repo_key, limit=99999, filter_str=None):
         :param filter_str:  OPTIONAL: users filter string
         :return:
         """
-        if not self.cloud:
-            url = 'rest/keys/1.0/projects/{project_key}/repos/{repo_key}/ssh'.format(project_key=project_key,
-                                                                                     repo_key=repo_key)
-        else:
-            url = 'rest/keys/2.0/projects/{project_key}/repos/{repo_key}/ssh'.format(project_key=project_key,
-                                                                                     repo_key=repo_key)
+        resource = self.resource_url('projects/{project_key}/repos/{repo_key}/ssh'.format(project_key=project_key,
```

Comment:
Also .../rest/keys/... and not .../rest/api/...

Reply:
@Spacetown Any good idea around that?

Reply:
thx, I just missed this part!

Reply:
Create a member function ressource_url_keys which does mealy the same as the ressource_url. I'll create a PR for this.